README.md

```diff
@@ -60,6 +60,7 @@
 ### Key Features <a name="key-features"></a>
 
 - **[Project Structure]**
+- **[Consume the api]**
```

Comment:
- Kindly, your `MIT` link has been broken, so please fix it to make the readme more professional 👍
<img width="909" alt="Screenshot 2023-05-17 142724" src="https://github.com/danielsafari143/leaderborad/assets/52010789/0dbd70de-f01d-40e6-a954-f4a8526c7a18">

Reply:
This change has been done well